docs/client/apidocs.rst

```diff
@@ -0,0 +1,110 @@
+API Reference
+=============
+
+Module contents
+---------------
+
+.. automodule:: scrapinghub.client
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+
+Submodules
+----------
+
+activity
+--------
```

Comment:
or, if you want a flat structure, follow an approach taken here http://flask.pocoo.org/docs/0.12/api/


```
API Reference
=============

Client object
-------------

Activity
--------

Collections
-----------

etc...
```

without splitting into `Module contents` and `Submodules`